src/Refitter.Core/ParameterExtractor.cs

```diff
@@ -96,9 +97,16 @@ private static string GetQueryAttribute(CSharpParameterModel parameter, RefitGen
     {
         return (parameter, settings) switch
         {
-            { parameter.IsArray: true } => "Query(CollectionFormat.Multi)",
-            { parameter.IsDate: true, settings.UseIsoDateFormat: true } => "Query(Format = \"yyyy-MM-dd\")",
-            { parameter.IsDate: true, settings.CodeGeneratorSettings.DateFormat: not null } => $"Query(Format = \"{settings.CodeGeneratorSettings?.DateFormat}\")",
+            { parameter.IsArray: true }
+                => "Query(CollectionFormat.Multi)",
+            { parameter.IsDate: true, settings.UseIsoDateFormat: true }
+                => "Query(Format = \"yyyy-MM-dd\")",
+            { parameter.IsDate: true, settings.CodeGeneratorSettings.DateFormat: not null }
```

Comment:
> If there is only Date and DateTime I see less use cases where you would want them diffrently. and if you want to have 2 different formats you can just make both in `CodeGenerationSettings`.

@Fargekritt That's true. Both `IsDate` and `IsDateOrTime` are both set to `true` for the `date` and `date-time` OpenAPI string formats

The only thing this pull request introduces is the ability to control how `date-time` formatting is handled explicitly